modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/metadata/OutputResolverFactory.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.runtime.module.extension.internal.metadata;
+
+import static java.lang.String.format;
+import static org.mule.runtime.api.util.Preconditions.checkArgument;
+
+import org.mule.runtime.api.metadata.resolving.OutputTypeResolver;
+import org.mule.sdk.api.metadata.NullMetadataResolver;
+
+/**
+ * Factory of {@link OutputTypeResolver}
+ */
```

Comment:
add since version

Reply:
Actually I shouldn't have included this class